Dart/src/com/jetbrains/lang/dart/flutter/FlutterUtil.java

```diff
@@ -38,4 +48,60 @@ public static boolean isFlutterModule(@NotNull final Module module) {
   public static boolean isFlutterPluginInstalled() {
     return FLUTTER_PLUGIN_INSTALLED;
   }
+
+  /**
+   * Returns true if passed pubspec declares a flutter dependency.
```

Comment:
=>

Returns true if passed a pubspec that declares a flutter dependency.

?

Reply:
modified